ui/src/utils/ajax.js

```diff
@@ -47,7 +47,7 @@ export default async function AJAX({
     }
   } catch (error) {
     const {response} = error
-    if (!response.status === RES_UNAUTHORIZED) {
+    if (!response.status === HTTP_UNAUTHORIZED) {
```

Comment:
Nope, different part of the codebase. It's confusing that I included that slight refactor in this commit - sorry about that!